physicalplan/plan.go

```diff
@@ -121,18 +177,19 @@ func newOperator(expr parser.Expr, storage storage.Queryable, mint time.Time, ma
 		return newCancellableOperator(e.Expr, storage, mint, maxt, step, lookbackDelta)
 
 	case *parser.StringLiteral:
+		// TODO(saswatamcode): Ensure Explain does not panic.
```

Comment:
Let's maybe remove handling of `StringLiteral` to resolve the panic. We should stabilize what we have now so that we can run the engine in actual environments.

Reply:
Will remove!

Reply:
Have removed the string literal support that we had previously for now. This would likely need some new operator!